src/lib/types.ts

```diff
@@ -368,3 +364,9 @@ export interface TimelyBulkUpdateEventsReturn {
 }
 
 export type TimelyEventBulkUpdate = OptionalExceptFor<TimelyEvent, 'id'>
+
+export type AddTimelyClient = OptionalExceptFor<TimelyClient, 'name'>
+
+export type AddTimelyUser = OptionalExceptFor<TimelyUser, 'name' | 'email' | 'role_id'>
+
+export type AddTimelyLabel = OptionalExceptFor<TimelyLabel, 'name'>
```

Comment:
Are there situations that result in Timely not responding with keys other the ones listed on create? Seems odd for them to not reply back with the full object.

Reply:
@CalebAlbers, just fixed this, I accidentally put those new Add Types as what's returned instead of as parameters